lib/ui/platform_dispatcher.dart

```diff
@@ -1316,7 +1323,7 @@ class ViewConfiguration {
     List<DisplayFeature>? displayFeatures,
   }) {
     return ViewConfiguration(
```

Comment:
should probably assert that at least one of window, view is null

Reply:
Added 7d2ad3ebb21e9b735bc20810991044fe34b72159.